polaris-react/src/components/Filters/Filters.scss

```
@@ -1,295 +1,261 @@
-    bottom: 0;
+// stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+@media #{$p-breakpoints-md-down} {
+  .FiltersWrapperWithAddButton {
     // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
     position: relative;
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    left: var(--p-space-4);
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    width: calc(100% - var(--p-space-8));
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    height: var(--p-space-025);
-    background-color: var(--p-surface-subdued);
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    display: block;
-  }
 
-  &::before {
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    display: none;
+    .FiltersInner {
+      padding-top: calc(var(--p-space-2) + var(--p-space-025));
+      padding-bottom: var(--p-space-5);
+      padding-right: 0;
+    }
   }
-}
 
-.open ~ .open {
-  &::before {
+  .AddFilterActivatorMultiple {
     // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    display: none;
-  }
-}
-
-.open.last {
-  &::before {
-    content: '';
+    position: sticky;
+    z-index: var(--p-z-1);
     // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
     top: 0;
     // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    position: relative;
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    left: var(--p-space-4);
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    width: calc(100% - var(--p-space-8));
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    height: var(--p-space-025);
-    background-color: var(--p-surface-subdued);
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    display: block;
-  }
-
-  &::after {
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    display: none;
-  }
-}
-
-.open + .last {
-  &::before {
+    right: 0;
     // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    display: none;
+    display: flex;
+    padding: var(--p-space-1) var(--p-space-4) var(--p-space-1) 0;
+    background: var(--p-surface);
+    margin-left: var(--p-space-2);
+
+    &::before {
+      content: '';
+      // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+      position: absolute;
+      // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+      top: 0;
+      // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+      left: -12px;
+      // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+      width: var(--p-space-3);
```

```suggestion
      width: 12px;
```

(Use hard coded values for layout declarations like width and height)